test/integration/task/analytics.js

```diff
@@ -77,5 +77,14 @@ describe('task: analytics', () => {
           // eslint-disable-next-line object-curly-spacing
           au.details.error.should.eql({ testError: 'foo'});
         }))));
+
+
+  it('should check xml content of what analytics reporter sent', testTask(async ({ Configs, analyticsReporter }) => {
+    await Configs.set('analytics', { enabled: true, email: 'test@getodk.org' });
```

Comment:
Is it worth adding `organization` here as well? Or are you intentionally testing that `organization` won't show up in `<config>` in the XML given that it wasn't specified? Anyway, I know this is a new/extra test, I was just wondering about these things.